src/main/java/io/github/balconyseats/vertx/helper/database/sql/ConfigSqlPoolInitializationContextHandler.java

```diff
@@ -0,0 +1,45 @@
+package io.github.balconyseats.vertx.helper.database.sql;
+
+import io.github.balconyseats.vertx.helper.application.InitializationContext;
+import io.github.balconyseats.vertx.helper.application.configurer.InitializationHandler;
+import io.vertx.core.Future;
+import io.vertx.core.Promise;
+import io.vertx.core.Vertx;
+import io.vertx.core.json.JsonObject;
+
+/**
+ * Creates {@link io.vertx.sqlclient.Pool} from configuration and
+ * add it to initialization context under "sqlpool" key
```

Comment:
add**s**

Missing dot at the end of the sentence.

Reply:
Fixed.